lib/index.js

```diff
@@ -138,13 +138,15 @@ module.exports = Serializer => {
 
       contextResponse.payload = {
         [reservedKeys.jsonapi]: jsonapi,
+        [reservedKeys.meta]: {},
         [reservedKeys.links]: {}
       }
 
       for (let type in recordTypes) {
         if (inflectType) type = inflection.transform(type, typeInflections[1])
-        contextResponse.payload[reservedKeys.links][type] = prefix +
-          uriTemplate.fillFromObject({ type })
+        contextResponse.payload[reservedKeys.links][type] = {
+          href: prefix + uriTemplate.fillFromObject({ type })
+        }
```

Comment:
The "links object" definition can be seen [here](http://jsonapi.org/format/#document-links).